### ISSUE TITLE
Enhanced softmax

### ISSUE THREAD

**Issue by sawatzkylindsey** [767-lvl/nn-wd#12]
* Show only the top K elements, according whatever the width of the neural network layers are
* Show the word labels themselves

If necessary, we needn't draw this in a box.
